components/scream/scripts/machines_specs.py

```diff
@@ -82,6 +82,20 @@
     "linux-generic-serial" : ([],["mpicxx","mpifort","mpicc"],"", get_cpu_core_count(), get_cpu_core_count(),""),
 }
 
+###############################################################################
```

Comment:
I think it would be easier to just modify MACHINE_METADATA directly if a local mach spec file is detected. Something like
```
MACHINE_METADATA = {...}
if pathlib.Path("~/.cime/scream_mach_specs.py").expanduser().is_file():
    sys.path.append(str(pathlib.Path("~/.cime").expanduser()))
    from scream_mach_specs import MACHINE_METADATA as LOCAL_MD
    if len(LOCAL_MD == 6):
        MACHINE_METADATA["local"] = LOCAL_MD
```

Then the rest of the file can remain unchanged.

Reply:
good idea. it will keep the code simpler in the rest of the file.